bin/tflocal

```diff
@@ -243,6 +249,28 @@ def get_region() -> str:
     return region or DEFAULT_REGION
 
 
+def get_access_key(provider=None) -> str:
+    access_key = str(os.environ.get("AWS_ACCESS_KEY_ID") or provider.get("access_key", "")).strip()
+    if access_key and access_key != DEFAULT_ACCESS_KEY:
+        # Change live access key to mocked one
+        return deactivate_access_key(access_key)
+    try:
+        # If boto3 is installed, try to get the access_key from local credentials.
+        # Note that boto3 is currently not included in the dependencies, to
+        # keep the library lightweight.
+        import boto3
+        profile = str(os.environ.get("AWS_PROFILE") or os.environ.get("AWS_DEFAULT_PROFILE", "default")).strip()
```

Comment:
Should we leave the profile handling to botocore, instead of handling the cases directly here? Seems like both cases are already covered in the internal implementation here: https://github.com/boto/botocore/blob/9b401c75699033b9988c03bcd51f221751087c5c/botocore/configprovider.py#L54

I think we could get away with simply creating a boto3 session with default configs like below, or do you see an edge case where this would not achieve the desired behavior?
```
access_key = boto3.session.Session().get_credentials().access_key
```